ProcessLib/HeatTransportBHE/CreateHeatTransportBHEProcess.cpp

```diff
@@ -19,6 +19,8 @@
 #include "HeatTransportBHEProcess.h"
 #include "HeatTransportBHEProcessData.h"
 
+#include <pybind11/pybind11.h>
```

Comment:
The process should not depend on python, in my opinion. Why is the dependency necessary and can it be moved in the boundary condition?

Reply:
This dependency cannot be moved to the boundary condition class. The reason is, the OGS-TESPy coupling is not only executed on the boundary condition level. At the end of each linear solver execution, inside of the HeatTransportBHEProcess::postIterationConcreteProcess function, the Python script will compare the simulated temperatures of each BHE and decide whether the OGS iteration should continue or not. This means, the process itself needs to be linked to pybind11 library.

Reply:
@HBShaoUFZ I have disabled building the process when `OGS_USE_PYTHON=OFF`. This fixed the failing Jenkins jobs.